test/e2e/case25_related_object_metric_test.go

```diff
@@ -58,18 +57,26 @@ var _ = Describe("Test related object metrics", Ordered, func() {
 			"-f", policyYaml,
 			"-n", testNamespace, "--ignore-not-found")
 		_, _ = cmd.CombinedOutput()
-		opt := metav1.ListOptions{}
-		utils.ListWithTimeout(
-			clientManagedDynamic, gvrConfigPolicy, opt, 0, false, defaultTimeoutSeconds)
+		utils.GetWithTimeout(
+			clientManagedDynamic, gvrConfigPolicy, policy1Name, testNamespace, false, defaultTimeoutSeconds,
+		)
+		utils.GetWithTimeout(
+			clientManagedDynamic, gvrConfigPolicy, policy2Name, testNamespace, false, defaultTimeoutSeconds,
+		)
 		utils.GetWithTimeout(
 			clientManagedDynamic, gvrConfigMap, relatedObject, "default", false, defaultTimeoutSeconds)
 	}
 
-	It("should clean up", cleanup)
-	It("should have no common related object metrics after clean up", func() {
```

Comment:
I don't think these lines should be removed. Adding an `AfterAll()` makes sense so that the cleanup always happens, but the cleanup before checking for metrics was intentional and part of the test.

Reply:
Thanks!!